yt_dlp/extractor/sbs.py

```diff
@@ -1,7 +1,13 @@
 from .common import InfoExtractor
 from ..utils import (
-    smuggle_url,
-    ExtractorError,
+    float_or_none,
+    HEADRequest,
```

Comment:
sort class above function
```suggestion
    HEADRequest,
    float_or_none,
```